docs/conversion/sakai_21_mysql_conversion.sql

```diff
@@ -4,3 +4,6 @@ DELETE SAKAI_MESSAGE_BUNDLE from SAKAI_MESSAGE_BUNDLE where PROP_VALUE is NULL;
 -- this constraint may have been missed, it is ok if this line fails just comment it out
 ALTER TABLE CONTENTREVIEW_ITEM ADD CONSTRAINT UK_8dngr1v68kkv4u11c1nvrjj1l UNIQUE (PROVIDERID, CONTENTID);
 
+-- SAK-30079
+ALTER TABLE MFR_PRIVATE_FORUM_T MODIFY COLUMN AUTO_FORWARD INT NOT NULL DEFAULT 2;
```

Comment:
how about TINYINT(1)

Reply:
AutoDDL creates it as INT apparently, wanted to keep it consistent. I agree it is wasteful when the possible values are only 0-2. I can change it if you want.

Reply:
i dont think it matters much but tinyint(1) is how it was created in the past 

msgcntr/messageforums-app/src/sql/mysql/sakai_messageForums.sql

Reply:
Yeah I saw that but I think that is from back when it was a boolean value and MySQL didn't have the BIT column type.

Reply:
I think we could possibly run into JDBC issues with tinyint(1) here as it appears that by default Connector/J treats tinyint(1) like bit.

https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-configuration-properties.html

Reply:
okay sounds like a future nightmare to unwind. i'm sure we have tons of tinyints all over the place